classes/class-ed-lti.php

```diff
@@ -58,11 +48,11 @@ public static function activate() {
 	 *
 	 * @return DataConnector
 	 */
-	private function lti_get_db_connector() {
+	private function get_db_connector() {
 		// phpcs:disable
 		return DataConnector::getDataConnector(
 			$this->wpdb->base_prefix,
-			new PDO( 'mysql:host=' . DB_HOST . ';dbname=' . DB_NAME, DB_USER, DB_PASSWORD )
+			new \PDO( 'mysql:host=' . DB_HOST . ';dbname=' . DB_NAME, DB_USER, DB_PASSWORD )
```

Comment:
We should add a `use PDO` statement at the top of the file and remove the backslash here.

Reply:
Hey Richard. I think this was missed in your initial pass at this. Could you take another look please? Many thanks

Reply:
@richeduni please could you address this comment? Thank you :)